### PR TITLE
Feat: move episode page to svelte

### PR DESCRIPTION
**Pull request by tokitouq**

### CODE REVIEW

File: backend/django_core/static_src/components/page/anime/episode/Index.svelte [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+                        <div class="accordion-summary flex-1">Monotone/Colorful</div>
+                        <div class="accordion-summary-caret fill-current w-3 transition-transform duration-[200ms] md:w-[1.25vw] rotate-180">
+                            <Chevron />
+                        </div>
+                    </button>
+                    <div class="accordion-panel  p-0 rounded-container-token text-surface-50 text-sm leading-snug md:text-[1vw] md:leading-[1.35vw]" id="accordion-panel-0.4186117384197431" role="region" aria-hidden="false" aria-labelledby="accordion-control-0.4186117384197431">The autumn he was twelve, piano prodigy Kousei Arima suddenly found himself unable to play the piano after his mother's death. Ever since then, it's like he's been frozen in time. His childhood friend, Tsubaki Sawabe, watches over him with concern; one day, she invites him on a double date. Kousei's other childhood friend, Ryouta Watari, is being introduced to a certain girl. Kousei reluctantly heads over to the rendezvous spot. There, he sees a girl playing a melodica. This girl, who allegedly has a crush on Watari, is Kaori Miyazono. And she turns out to be a violinist!</div>
```
Comment (baseplate-admin) [inline backend/django_core/static_src/components/page/anime/episode/Index.svelte:108]: Isn't this dynamic?

File: backend/django_core/static_src/components/page/anime/episode/Index.svelte [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                        loading="lazy"
+                    >
+                </div>
+            </div>
+            <div class="flex flex-col gap-2 px-5 md:flex-row md:items-center md:justify-between md:gap-0 md:p-0">
+                <div class="flex gap-2 md:items-center md:gap-[1vw]">
+                    <div class="hidden items-center gap-[0.75vw] md:flex">
+                        <span class="text-[1vw] font-semibold uppercase">sub/dub:</span>
+                        <button class="btn !bg-transparent min-h-full flex items-center gap-[0.5vw] rounded-[0.35vw] px-[0.75vw] py-[0.5vw] text-[1vw] leading-none">
+                            Vidstreaming (sub)
```
Comment (baseplate-admin) [inline backend/django_core/static_src/components/page/anime/episode/Index.svelte:34]: Dynamic?

File: backend/django_core/static_src/components/page/anime/episode/Index.svelte (hunk 1 above)
Comment (tokitouq) [reply backend/django_core/static_src/components/page/anime/episode/Index.svelte:108]: XD no? its not

File: backend/django_core/static_src/components/page/anime/episode/Index.svelte (hunk 2 above)
Comment (tokitouq) [reply backend/django_core/static_src/components/page/anime/episode/Index.svelte:34]: .... nah, you messed up

**Pull request merged**